src/js/pouch.js

```diff
@@ -1,21 +1,42 @@
-// Utility functions to add pouch to an existing express instance
+// Provide a CouchDB-like API using `PouchDB` and `express-pouchdb`.  This in only useful in association with an
+// existing `gpii.express` instance.
+//
+// The "databases" option is expected to be an array keyed by dbName, with options to control whether data is loaded or
+// not, as in:
+//
+//  databases: {
+//    "nodata": {},
+//    "data":   { "data": "../tests/data/records.json" }
+//  }
+//
+// In this example, an empty database called "nodata" would be created, and a database called "data" would be created
+// and populated with the contents of "../tests/data/records.json".
+//
+// NOTE:
+//   This module has a serious and non-obvious limitation, in that only one instance of PouchDB is created.  This
```

Comment:
Looks like this comment should be updated in the light of the new "cleaning" facility